src/shared/components/button-tab/ButtonTabGroup.tsx

```
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+import ButtonTab from './ButtonTab';
+
+interface Tab {
+  id: string
+  label: string
+}
+
+interface ButtonTabGroupProps {
+  tabs: Tab[]
+  defaultActiveId?: string
+  handleTabChange?: (tabId: string) => void
```

ESLint

[error] 'tabId' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)